README.md

```diff
@@ -656,7 +683,7 @@ These are all the code changes! 🎉
 
 Learn how to configure AcraCensor – SQL firewall – to allow or deny specific queries and make your application more steady against SQL injections.
 
-What's inside: OWASP Mutillidae vulnerable web application, AcraConnector, AcraServer, AcraCensor.
+What's inside: OWASP Mutillidae vulnerable web application, AcraServer, AcraCensor.
```

Comment:
have you updated our example with acracensor as well?

Reply:
no. returned connector for connector